app/assets/stylesheets/sass/buttons.scss

```diff
@@ -50,3 +51,6 @@ input.search-crud-buttons {
     @include buttonhovers(purple);
   }
 }
+input.vote-button {
+  @include buttonstyles(#888);
```

Comment:
Color literals like `#888` should only be used in variable declarations; they should be referred to via variable everywhere else.